jupyter_book/commands/__init__.py

```diff
@@ -67,37 +75,58 @@ def build(path_book, path_output, config, toc, warningiserror, build):
     if config is not None:
         book_config["yaml_config_path"] = str(config)
         config_yaml = yaml.safe_load(config.read_text())
-
         # Pop the extra extensions since we need to append, not replace
         extra_extensions = config_yaml.pop("sphinx", {}).get("extra_extensions")
+        # Support Top Level config Passthrough
+        # https://www.sphinx-doc.org/en/latest/usage/configuration.html#project-information
+        sphinx_options = ["project", "author", "copyright"]
+        for option in sphinx_options:
+            if option in config_yaml.keys():
+                book_config[option] = config_yaml[option]
 
     # Builder-specific overrides
-    if build == "pdf_html":
-        theme_options = book_config.get("html_theme_options", {})
-        theme_options["single_page"] = True
-        book_config["html_theme_options"] = theme_options
-
-    OUTPUT_PATH = path_output if path_output is not None else PATH_BOOK
-    OUTPUT_PATH = Path(OUTPUT_PATH).joinpath("_build/html")
+    latex_config = None
+    if builder == "pdfhtml":
+        book_config["html_theme_options"] = {"single_page": True}
+    if builder == "pdflatex":
+        if "latex" in config_yaml.keys():
+            latex_config = config_yaml.pop("latex")
```

Comment:
will there potentially be other keys in `latex:` that aren't direct latex pass-throughs? If so, perhaps we should use a structure like this:

```yaml
latex:
  ourcustomoption1: val1
  config:
    passthroughkey: passthroughval
```

but maybe it's unnecessary if we don't imagine any other kind of latex config than what we'd pass through

Reply:
hey @choldgraf not sure if we will need much config that doesn't get passed through to sphinx. What might that be? I am inclined to have a top level `latex` section and then filter the sphinx options through to sphinx -- but I guess that means maintaining an api and adding sphinx options via a filter.